File: app/src/main/java/com/android/virtus/chatapp/MainActivity.java
package com.android.virtus.chatapp;

import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    public static int DELAY_TIME = 1000;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        new Handler().postDelayed(new Runnable() {
            @Override
            public void run() {
                Intent itDangNhap = new Intent(MainActivity.this,DangNhapActivity.class);
                startActivity(itDangNhap);
                finish();
            }
        },DELAY_TIME);
    }
}

File: app/src/main/java/com/android/virtus/chatapp/MainActivity.kt
package com.android.virtus.chatapp

import android.content.Intent
import android.os.Bundle
import android.os.Handler
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        Handler().postDelayed({
            val itDangNhap = Intent(this@MainActivity, DangNhapActivity::class.java)
            startActivity(itDangNhap)
            finish()
        }, DELAY_TIME.toLong())
    }

    companion object {
        var DELAY_TIME = 1000
    }
}